ConsistencyManager/ConsistencyManager.swift

```diff
@@ -95,6 +95,12 @@ open class ConsistencyManager {
     var listeners = [String: WeakListenerArray]()
 
     /**
+     This is an array of all the model update listeners.
+     These will get notified whenever anything changes in the Consistency Manager.
+     */
```

Comment:
Yes. Do you get why this is useful given https://github.com/linkedin/ConsistencyManager-iOS/issues/45